### ISSUE TITLE
What's is the V5 colunmn of GWAS summary statistics in your vignette 

### ISSUE THREAD

**Issue by aina91** [caleblareau/gchromVAR#10]
Hi, I am interetsed in your work. Excuse me, what does the fifth column of GWAS summary statistics files in vigente represent？
It shows below.

## 6 chr1 25095653 25095654 region1 0.0061

And I also want to known how to make this format GWAS summary statistics files. Is there a script shipped with gchromVAR that does this, or do I just  make it by my self?Thanks!

**Comment by julirsch** [comment]
This is the posterior inclusion probability from fine-mapping. Unfortunately you have to perform fine-mapping yourself.

**Comment by aina91** [comment]
Yeah,I see. Is there any partial scripts in github which are correlated to 'Genome-wide association studies','Linkage disequilibrium score regression', to get input files ( master files, z files and ld files) of Fine_mapping. It will be very helpful for me to understand and use 'g-chromVAR'.



At 2020-09-04 23:34:11, "Jacob Ulirsch" <notifications@github.com> wrote:

This is the posterior inclusion probability from fine-mapping. Unfortunately you have to perform fine-mapping yourself.

**Comment by julirsch** [comment]
I don't think so. There is a nice and clear wiki for how to use LDSC here:
https://github.com/bulik/ldsc/wiki

We essentially just used Unix commands to munge output from BOLT-LMM and FINEMAP, both of which are well documented below:
https://storage.googleapis.com/broad-alkesgroup-public/BOLT-LMM/BOLT-LMM_manual.html
http://www.christianbenner.com
